src/registrar/assets/sass/_theme/_tables.scss

```diff
@@ -26,6 +26,18 @@
     padding-bottom: units(2px);
   }
 
+  td {
```

Comment:
You can get rid of the wrapping td. Less nesting is both more efficient and more reusable.

Reply:
That is interesting how that is more performant, I would assume it narrows the search space? Neat, thanks for sharing